### PR TITLE
Jose and Hannah

### PR DESCRIPTION
**Pull request by Astronaut**

### CODE REVIEW

File: assignment/scripts/describe.js [hunk 1]
```diff
@@ -26,10 +26,9 @@ if (number >= 2) {
 
 
 // 1. WRITE YOUR DESCRIPTION HERE
-// 
-// 
-// 
-//
+// We make a variable called name and set it as Dane
+// We check if the name is equal to Mary.
+// Dane is not equal to Mary, so we console log 'How do you do?'
```
Comment (delventhalz) [inline assignment/scripts/describe.js:31]: What would the code do if the name *was* equal to Mary?

File: assignment/scripts/describe.js [hunk 2]
```diff
@@ -70,10 +71,15 @@ console.log(secret)
...
-// 
-// 
-//
+// We set a variable called isStudent and set it to "True" as a boolean.
+// We also set a variable called age and set it to "34" as a number.
+// We also set a variable called zip and set it to "55407" as a number.
+// We check if isStudent is True and if zip is greater then 80000, which is not, so we skip. 
+// We check if isStudent is False or age is less than 30, which is not, so we skip.
+// We check if isStudent is equal to True, which it is, so we console log 'Welcome to Prime!'
+// The last else conditional is skipped since we met a condition.
```
Comment (delventhalz) [inline assignment/scripts/describe.js:80]: Similarly, I would love to see more description of what the logic of the code does, not just what it does in the case where the variables are set up the way they happen to be set up.

File: assignment/scripts/describe.js [hunk 3]
```diff
@@ -147,6 +159,8 @@ if (mix === true) {
 let temp = 40;
 const time = 4;
 
+// need to change or to and which is && in the if statement
```
Comment (delventhalz) [inline assignment/scripts/describe.js:162]: Wording here is confusing

File: assignment/scripts/describe.js (hunk 1 above)
Comment (Astronaut) [reply assignment/scripts/describe.js:31]: If the name were equal to Mary, the code would console log "Hi, Mary!"

**Pull request closed without merging**